runtime/browser/xwalk_content_browser_client.cc

```diff
@@ -430,7 +430,7 @@ XWalkContentBrowserClient::CreateThrottlesForNavigation(
         navigation_interception::InterceptNavigationDelegate::CreateThrottleFor(
             navigation_handle));
   }
-  return std::move(throttles);
+  return throttles;
```

Comment:
How do you know it is preventing copy elision? Does compiler throw any error or warning message?